pwndbg/disasm/__init__.py

```diff
@@ -100,11 +100,13 @@ def get_one_instruction(address):
         pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
         return ins
 
-def one(address=None):
+
+
+def one(address='unset_argument'):
+    if address == 'unset_argument':
```

Comment:
if cause pwndbg.regs.pc, that waht the second condition is supposed to do but it got taked to absurdum by returning before the assignment in case address is None.
If you fix it and set address to pwndbg.regs.pc once the argument is None, the whole disassembly will go nuts.